src/main/scripts/src/app/shared/search-box/search-box.component.html

```diff
@@ -12,6 +12,6 @@
   </div>
 </div>
 
-<button (click)="onButtonClick()">
+<button class="fa fa-search fa-fw" (click)="onButtonClick()">
```

Comment:
Why not using the default font-awesome way of adding icons to your code?
```html
<i class="fa fa-search" aria-hidden="true"></i>
```
You can add this inside the `<button>` element just before `<span>`.